src/pages/en/guides/markdown-content.md

```diff
@@ -357,6 +357,24 @@ const { Content } = Astro.props.post
   <Content/>
 </article>
 ```
+## Fetching Remote Markdown
+
+Astro was designed to render Markdown to HTML as a part of building your website. However, there may be certain cases where you need to fetch Markdown from a remote source at runtime, after the site has been built. For example, you may want to fetch and render Markdown from a remote API every time a user makes a request to your website.
+
+**Astro does not include a runtime Markdown parser!** To render remote Markdown to HTML at runtime, you will need to install and configure your own Markdown parser. This **will not** inherit from any of Astro's built-in Markdown and MDX settings that you have configured. Be sure that you understand these limitations before implementing this in your project. 
```

Comment:
This description relies heavily on the concept of a “runtime” as a way to flag _when_ something happens. Looking through docs we use this in a few different contexts but I don’t think we ever really explain what this jargon means.

For example, on Getting Started we use it to mean both client-side and server-side JS:

> - **Zero JS, by default:** No JavaScript **_runtime_** overhead to slow you down.
> - **Edge-ready:** Deploy anywhere, even a global edge **_runtime_** like Deno or Cloudflare.

I think “runtime” is a bit vague because you also _run_ your build, which I guess you could call “Astro runtime”.

Additionally, although currently this section is focused on SSR, this is equally valid/helpful advice for remote Markdown sources a user might want to compile at build time, so we can probably generalise this.

```suggestion
Astro was designed to render local Markdown files to HTML when building your website. However, there may be certain cases where you need to fetch Markdown from a remote source. For example, you may want to fetch and render Markdown from a remote API at build time or, if using [SSR](/en/guides/server-side-rendering/), every time a user makes a request to your website.

To render remote Markdown to HTML, you will need to install and configure your own Markdown parser. This **will not** inherit from any of Astro's built-in Markdown and MDX settings that you have configured. Be sure that you understand these limitations before implementing this in your project. 
```

How does this sound?